front/src/Phaser/Game/GameMap.ts

```diff
@@ -424,16 +421,16 @@ export class GameMap {
     }
 
     public getArea(name: string): ITiledMapRectangleObject | undefined {
-        return this.areas.get(name);
+        return this.areas.find((area) => (area.name = name));
```

Comment:
```suggestion
        return this.areas.find((area) => (area.name === name));
```
I'm pretty sure the === is what you meant :)